proto/cosmos/authz/v1beta1/authz.proto

```diff
@@ -22,8 +22,11 @@ message GenericAuthorization {
 // Grant gives permissions to execute
 // the provide method with expiration time.
 message Grant {
-  google.protobuf.Any       authorization = 1 [(cosmos_proto.accepts_interface) = "Authorization"];
-  google.protobuf.Timestamp expiration    = 2 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
+  google.protobuf.Any authorization = 1 [(cosmos_proto.accepts_interface) = "Authorization"];
+  // time when the grant will expire and will be pruned. If null, then the grant
+  // doesn't have a time expiration (other conditions  in `authorization`
+  // may apply to invalidate the grant)
+  google.protobuf.Timestamp expiration = 2 [(gogoproto.stdtime) = true, (gogoproto.nullable) = true];
```

Comment:
```suggestion
  google.protobuf.Timestamp expiration = 2 [(gogoproto.stdtime) = true];
```
I think it is not mandatory to mention `(gogoproto.nullable) = true` for nullable fields.

Reply:
correct, I left it for documentation purpose. But it's already explained in a comment. Do you think it's better to remove it?

Reply:
Yes it's not needed, we haven't mentioned in feegrant too.